storage/persistent_storage/rosa-persistent-storage-aws-ebs.adoc

```diff
@@ -0,0 +1,70 @@
+:_content-type: ASSEMBLY
+[id="rosa-persistent-storage-aws-ebs"]
+= Persistent storage using AWS Elastic Block Store (EBS)
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: rosa-persistent-storage-aws-ebs
+
+toc::[]
+
+{product-title} (ROSA) clusters are pre-built with two storage classes that use AWS Elastic Block Store (EBS) volumes. These storage classes are ready to use and some familiarity with Kubernetes and AWS is assumed.
+
+Following are the two pre-built storage classes:
+[options="header"]
+
+|===
+
+| Name | Provisioner
+
+| gp2 | kubernetes.io/aws-ebs
+
+| gp2-csi | ebs.csi.aws.com
+
+|===
+
+[NOTE]
+====
+The gp2 storage class is set as default; however, you can select either one as the default storage class.
+====
+
+The Kubernetes persistent volume framework enables the administrators to provision a cluster with persistent storage and gives users a way to request those resources without having any knowledge of the underlying infrastructure. You can dynamically provision the AWS EBS volumes. Persistent volumes are not bound to a single project or namespace; therefore, the volumes can be shared across ROSA clusters. Persistent volume claims are specific to a project or namespace and can be requested by users.
+
+[IMPORTANT]
+====
+ROSA defaults to using an in-tree or non-Container Storage Interface (CSI) plug-in to provision AWS EBS storage. In future ROSA versions, volumes provisioned using existing in-tree plug-ins are planned for migration to their equivalent CSI driver. After full migration, the in-tree plug-ins will eventually be removed from the future versions of ROSA.
+====
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying storage provider.
+====
+
+== Format of persistent volumes
```

Comment:
This may be different for ROSA docs, but OCP core would require an [id] for each section. Same applies for `== Capacity of EBS volumes on a node`

Reply:
Added the [id] for each section, and I'll try using this as basic repo hygiene going forward. Thank you!